templates/internet-of-things/smart-city.html

```diff
@@ -0,0 +1,705 @@
+        ) | safe
+      }}
+    </div>
+    <div class="col-6">
+      <h2>Smart cities need strong IoT device security</h2>
+      <p class="p-heading--4">Strong security</p>
+      <p>Regardless of the smart city solution you are building, ensure you have the tools in place to guarantee a secure software environment.</p>
+      <p>Ubuntu Core offers secure boot, <a href="/engage/iot-disk-encryption">full disk encryption</a> and <a href="https://snapcraft.io/docs/assertions" class="p-link--external">signed software packages</a>, guaranteeing system integrity. <a href="https://snapcraft.io/docs/snap-confinement" class="p-link--external">Snap confinement</a> keeps your apps separated and controls access to hardware.</p>
+      <p>Curate your <a href="/internet-of-things/appstore">IoT App Store</a> from global suppliers, with constant application security scanning and updates.</p>
+      <p>
+        <a href="https://assets.ubuntu.com/v1/3d9fc6c3-A.Cybersecurity.with.Ubuntu_11.10.21.1.pdf" class="p-link--external">Cybersecurity with Ubuntu</a>
+      </p>
+    </div>
+  </div>
+
+  <div class="u-fixed-width">
+    <hr class="p-separator">
+  </div>
+
+  <div class="row">
+    <div class="col-6">
+      <h2>IoT OTA updates</h2>
+      <p class="p-heading--4">Compressed, transactional, signed and secure</p>
+      <p>Update any system, any device, any time. Future-proof your fleet with automatic, over-the-air updates.</p>
+      <p>With Ubuntu Core, full OS containerisation means transactional, atomic updates, data snapshot backup and restore, signed software integrity, <a href="/engage/snap-deltas">compressed delta updates</a> and more.</p>
+      <p><a href="/engage/snapd-whitepaper">Complete control</a>. No delayed patches, no partial updates. If for any reason an update fails, the device will roll back to its last stable state automatically.</p>
+      <p>
+        <a href="/core/features/ota-updates">OTA updates for Ubuntu Core&nbsp;&rsaquo;</a>
+      </p>
+    </div>  
+    <div class="col-6 u-vertically-center u-align--center u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/8998cab5-control-thin.svg",
+        alt="",
+        width="320",
+        height="320",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+
+  <div class="u-fixed-width">
+    <hr class="p-separator">
+  </div>
+
+  <div class="row">
+    <div class="col-6">
+      <div class="p-logo-section">
+        <div class="p-logo-section__items">
+          <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/f6a59799-partner-logo-arm.svg",
+            alt="",
+            width="112",
+            height="34",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+          </div>
+          <div class="p-logo-section__item">
+          {{ image (
```

Comment:
I can't see the intel logo on the demo for some reason.

Reply:
That's odd, I can see it. Could it be a caching issue?

Reply:
![image](https://user-images.githubusercontent.com/57550290/143888107-ffd91da8-bc14-450f-a3af-9f1cf11b0061.png)
Hmm thats weird. I can't see it in the incognito window as well.

Reply:
Oh Pete! The image is not loading on my laptop screen but I can see it on my external monitor when refreshing it. Maybe because the image is too big?

Reply:
I think you're right, I have reduced the size. Weird because the class of p-logo-section should handle the sizing.